File: show-case/mybatis-show-case/src/main/java/com/yicj/study/mybatis/HelloMybatisApplication.java
package com.yicj.study.mybatis;


import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@MapperScan("com.yicj.study.mybatis.mapper")
public class HelloMybatisApplication {

    public static void main(String[] args) {
        SpringApplication.run(HelloMybatisApplication.class, args) ;
    }
}

File: show-case/mybatis-show-case/src/main/java/com/yicj/study/mybatis/HelloMybatisApplication.kt
package com.yicj.study.mybatis

import com.yicj.study.mybatis.model.Person
import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
@MapperScan("com.yicj.study.mybatis.mapper")
open class HelloMybatisApplication

fun main(args: Array<String>) {
    SpringApplication.run(HelloMybatisApplication::class.java, *args)
}